be/src/olap/column_vector.cpp

```diff
@@ -224,6 +224,16 @@ Status ArrayColumnVectorBatch::resize(size_t new_cap) {
     return Status::OK();
 }
 
+void ArrayColumnVectorBatch::put_item_ordinal(segment_v2::ordinal_t* ordinals, size_t start_idx,
+                                              size_t size) {
+    size_t first_offset = *(_offsets->scalar_cell_ptr(start_idx));
+    segment_v2::ordinal_t first_ordinal = ordinals[0];
```

Comment:
need DCHECK(size > 0) or put this line inside while loop, do not worry about performance, the compiler will cover it.

Reply:
OK